src/pipdeptree/_models/package.py

```diff
@@ -23,13 +23,9 @@ class Package(ABC):
 
     UNKNOWN_LICENSE_STR = "(Unknown license)"
 
-    def __init__(self, obj: DistInfoDistribution) -> None:
-        self._obj: DistInfoDistribution = obj
-        self.key = pep503_normalize(obj.key)
-
-    @property
-    def project_name(self) -> str:
-        return self._obj.project_name  # type: ignore[no-any-return]
+    def __init__(self, obj: Distribution) -> None:
+        self._obj: Distribution = obj
+        self.editable = False
```

Comment:
I don't think we need this since we'll be passing a `DistPackage` to `FrozenRequirement.from_dist()`.

Reply:
fixed